frontend/src/components/messstelle/optionsmenue/panels/TagesTypRadiogroup.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <v-row no-gutters>
+        <v-col cols="6">
+            <v-radio-group
+                v-model="chosenOptionsCopy.tagesTyp"
+                class="full-width"
+            >
+                <v-radio
+                    :value="TagesTyp.WERKTAG_DI_MI_DO"
+                    :label="getTagesTypText(TagesTyp.WERKTAG_DI_MI_DO)"
+                />
+                <v-radio
+                    :value="TagesTyp.WERKTAG_MO_FR"
+                    :label="getTagesTypText(TagesTyp.WERKTAG_MO_FR)"
+                />
+                <v-radio
+                    :value="TagesTyp.SAMSTAG"
+                    :label="getTagesTypText(TagesTyp.SAMSTAG)"
+                />
+                <v-radio
+                    :value="TagesTyp.SONNTAG_FEIERTAG"
+                    :label="getTagesTypText(TagesTyp.SONNTAG_FEIERTAG)"
+                />
+                <v-radio
+                    :value="TagesTyp.WERKTAG_FERIEN"
+                    :label="getTagesTypText(TagesTyp.WERKTAG_FERIEN)"
+                />
+                <v-radio
+                    :value="TagesTyp.DTV"
+                    :label="getTagesTypText(TagesTyp.DTV)"
+                />
+            </v-radio-group>
+        </v-col>
+        <v-col cols="6"> {{ helperText }}</v-col>
```

Comment:
Text wird nicht angezeigt
![image](https://github.com/it-at-m/dave-frontend/assets/18263366/40846006-9e5e-436e-9534-11e1e2e12f1f)

Reply:
der text wird nur angezeigt, wenn es sich auch um einen Zeitraum handelt. Im folgeticket ist dann enthalten, dass die optionen hier nicht angezeigt werden wenn es nur ein einzelner tag ist. Hast du einen Zeitraum eingegeben und der text ist trotzdem nicht gekommen?

Reply:
Hatte es nur mit einem Einzeltag probiert